PerfNext/config/benchmarks/data_simple/Liberty.xml

```diff
@@ -14,22 +14,12 @@
 				<property name="MEASURE_TIME">180</property>
 				<property name="WARMUP_TIME">180</property>
 				<property name="RESULTS_MACHINE">lowry1</property>
-				<property name="RESULTS_DIR">libertyResults-cleaned</property>
-				<property name="CLIENT">perfxposh10G</property>
-				<property name="CLIENT_WORK_DIR">/java/perffarm/liberty</property>
-				<property name="DB_MACHINE">perfxposh10G</property>
-				<property name="DB2_HOME">/home/db2inst1/</property>
-				<property name="DB_NAME">day30r</property>
-				<property name="DB_PORT">50000</property>
-				<property name="DB_USR_NAME">db2inst1</property>
-				<property name="SERVER_NAME">DayTrader3-$JDK</property>
-				<property name="LIBERTY_HOST">kermit10G</property>
+				<property name="RESULTS_DIR">libertyResults-cleaned</property>			
+				<property name="SERVER_NAME">DayTrader3-$JDK</property>				
 				<property name="SECOND_CLIENT"></property>
```

Comment:
You can remove `SECOND_CLIENT` and `THIRD_CLIENT` since we don't use them. In case, we do need them in future, we can add them to the machine HW specific vars.